h/static/scripts/directive/test/annotation-test.js

```diff
@@ -1424,30 +1424,53 @@ describe('annotation', function() {
       });
     });
 
-    describe('annotation metadata', function () {
-      function findLink(directive) {
-        var links = directive.element[0]
-          .querySelectorAll('header .annotation-header__timestamp');
-        return links[links.length-1];
-      }
+    describe('annotation links', function () {
+      it('linkInContext uses the in-context links when available', function () {
+        var annotation = Object.assign({}, fixtures.defaultAnnotation(), {
+          links: {
+            html: 'https://test.hypothes.is/a/deadbeef',
+            incontext: 'https://hpt.is/deadbeef'
+          },
+        });
+        var controller = createDirective(annotation).controller;
+
+        assert.equal(controller.linkInContext, annotation.links.incontext);
+      });
 
-      it('displays HTML links when in-context links are not available', function () {
+      it('linkInContext falls back to the HTML link when in-context links are missing', function () {
         var annotation = Object.assign({}, fixtures.defaultAnnotation(), {
-          links: {html: 'https://test.hypothes.is/a/deadbeef'},
+          links: {
+            html: 'https://test.hypothes.is/a/deadbeef',
+          },
         });
-        var directive = createDirective(annotation);
-        assert.equal(findLink(directive).href, annotation.links.html);
+        var controller = createDirective(annotation).controller;
+
+        assert.equal(controller.linkInContext, annotation.links.html);
       });
 
-      it('displays in-context links when available', function () {
+      it('linkHTML uses the HTML link when available', function () {
         var annotation = Object.assign({}, fixtures.defaultAnnotation(), {
           links: {
             html: 'https://test.hypothes.is/a/deadbeef',
-            incontext: 'https://hpt.is/deadbeef'
```

Comment:
Do we want to test that `linkHTML` is always the HTML link, even when `incontext` is available?